internal/auth/certs.go

```diff
@@ -1,5 +1,3 @@
-// Package auth collects structures and functions around the
```

Comment:
Is this not required ?

Reply:
No. The description is at the top of `auth.go`. Repeating it here causes the docs generator to repeat it as well in the output.